ixdiagnose/plugins/dbdump.py

```diff
@@ -0,0 +1,13 @@
+from ixdiagnose.utils.command import Command
```

Comment:
Please remove this module. Though it exists currently, this isn't actually called by default because we had a bunch of enterprise paying customers (as well as end-users) not like the fact that their entire db contents were dumped to a file.